README.md

```diff
@@ -1,2 +1,32 @@
-# uprotocol-platform-linux-zenoh
-Linux uPlatform that is based off the Zenoh uTransport
+# up-streamer-rust
+
+Generic, pluggable uStreamer that should be usable in most places we need
+to bridge from one transport to another.
+
+## Overview
+
+Implementation of the uProtocol's uStreamer specification in Rust.
+
+Documentation can be generated locally with:
+```bash
+cargo doc --package up-streamer --open
+```
+which will open your browser to view the docs.
+
+## Getting Started
+
+### Working with the library
+
+`up-streamer-rust` is generic and pluggable and can serve your needs so long as
+1. Each transport you want to bridge over has a `up-client-foo-rust` library
+   and UPClientFoo struct which has `impl`ed `UTransport`
+2. `UTransportBuilder` has been `impl`ed on a struct so that a 
+   `Box<dyn UTransport>` can be safely created by the `UTransportRouter`
+   in the proper thread's context
+
+### Usage
+
+After following along with the docs to add all your forwarding routes, you'll
```

Comment:
which docs are you referring to here?

Reply:
Ah, not clear perhaps.

The docs generated by `cargo doc` mentioned right above this:
```bash
cargo doc --package up-streamer --open
```